<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>order</class>
 <widget class="QWidget" name="order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="top_layout" stretch="0,0,0,1">
     <item>
      <layout class="QGridLayout" name="order_layout" columnstretch="0,0,1">
       <item row="0" column="0" rowspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Order Id</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>PO Number</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Ordered Date</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Processing</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QLineEdit" name="order_id">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2" rowspan="7">
        <widget class="QPlainTextEdit" name="notes"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="po_number"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="ordered_date">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Due Date</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="processing">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
         <item>
          <widget class="QLineEdit" name="priority_digit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="priority_slider">
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>555</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>true</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Priority</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="customer_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="customer_label">
         <property name="text">
          <string>Customer:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="customer"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTabWidget" name="tabs">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_shipping">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <attribute name="title">
           <string>Shipping</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="4" column="0" colspan="2">
            <widget class="QLineEdit" name="s_city">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="placeholderText">
              <string>City</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="s_state">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="placeholderText">
              <string>State</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>208</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="0">
            <widget class="QLineEdit" name="s_zipcode">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="placeholderText">
              <string>Zip Code</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QLineEdit" name="s_address2">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="placeholderText">
              <string>Street Address</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QComboBox" name="s_address1">
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_billing">
          <attribute name="title">
           <string>Billing</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="3" column="0" colspan="2">
            <widget class="QLineEdit" name="b_city">
             <property name="placeholderText">
              <string>City</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="b_state">
             <property name="placeholderText">
              <string>State</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>208</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="0">
            <widget class="QLineEdit" name="b_zipcode">
             <property name="placeholderText">
              <string>Zip Code</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QLineEdit" name="b_address2">
             <property name="placeholderText">
              <string>Street Address</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QComboBox" name="b_address1">
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="table_items"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="prices_layout" stretch="0,0,1,0,0,0,0">
     <item>
      <widget class="QLineEdit" name="part_number">
       <property name="placeholderText">
        <string>Part Number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_part">
       <property name="text">
        <string>Add Part</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>628</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="discount_label">
       <property name="text">
        <string>Discount</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="discount">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>00.00%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="order_total_label">
       <property name="text">
        <string>Order Total</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="order_total">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMask">
        <string>$#.#; </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_invoice">
   <property name="text">
    <string>Print Invoice</string>
   </property>
   <property name="toolTip">
    <string>Print Invoice</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>Delete Order</string>
   </property>
   <property name="toolTip">
    <string>Delete Current Order</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_list">
   <property name="text">
    <string>Print List</string>
   </property>
   <property name="toolTip">
    <string>Print List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="action_finished">
   <property name="text">
    <string>Finished</string>
   </property>
   <property name="toolTip">
    <string>Mark Order as Finished</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+`</string>
   </property>
  </action>
  <zorder>table_items</zorder>
  <zorder>priority_digit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
